src/main/java/com/meilisearch/sdk/Key.java

```diff
@@ -5,8 +5,6 @@
 
 /** MeiliSearch response for a Key */
 public class Key {
-    // This field has been set to "" pending resolution of this issue
-    // https://github.com/meilisearch/meilisearch/issues/2116
```

Comment:
Can you look if there are other occurrences of this?

Reply:
Your mean about this issue in this SDK or other issues in this SDK or this issue in the other SDK?

Reply:
Nope, I saw that you fixed the issue here https://github.com/meilisearch/meilisearch-java/pull/344 and then removed all the comments like this one, from the files, but you missed this. My question is, there are other comments about this subject lost somewhere?

Reply:
I hope not this time 😄